exo-create/helpers/helpers.go

```diff
@@ -0,0 +1,67 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+const projectJsonContent = `
+{
+  "AppName": "my-app",
+  "ExocomVersion": "0.22.1",
+  "AppVersion": "0.0.1",
+  "AppDescription": ""
+}
+`
+
+const applicationYmlContent = `name: {{AppName}}
```

Comment:
While this helps with readability of the code, I think it adds an empty line to the top of `application.yml` file. I don't know if this affects the code that reads the file.